spec/unit/comments_spec.rb

```diff
@@ -126,7 +126,7 @@
           namespace: namespace_name)
 
         expect(ActiveAdmin::Comment.find_for_resource_in_namespace(publisher, namespace_name).last.resource_type).
-          to eq('Publisher')
+          to eq('User')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.